### PR TITLE
Fix the spec build error

### LINKED ISSUE #754: [subresource-loading] Update the spec to reflect the upstream changes of HTML.
It seems the spec build is failing because there were some changes in upstream.
Probably that is https://github.com/whatwg/html/commit/b83a82ff1efc8d64d6ec54f93dc5163702c4e46f.

cc: @hiroshige-g

### PR DESCRIPTION
**Pull request by hayatoito**
Fix #754. Probably we need to refine more, but this would be enough to avoid spec build errors in CI.

***
<a href="https://pr-preview.s3.amazonaws.com/hayatoito/webpackage/pull/797.html" title="Last updated on Sep 28, 2022, 1:43 AM UTC (beaf329)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/webpackage/797/c614114...hayatoito:beaf329.html" title="Last updated on Sep 28, 2022, 1:43 AM UTC (beaf329)">Diff</a>

### CODE REVIEW

File: subresource-loading.bs [hunk 1]
```diff
@@ -130,37 +137,33 @@ ISSUE: Not supported for workers.
...
-Inside the <a spec="html">prepare a script</a> algorithm, we make the following
-changes:
+Inside the <a spec="html">prepare the script element</a> algorithm, we make the
+following changes:
 
-- Insert the following step to [=prepare a script=] step 8, under "Determine the
-  script's type as follows:":
-  - If the script block's type string is an [=ASCII case-insensitive=] match for
-    the string "`webbundle`", <a spec="html">the script's type</a> is
-    "`webbundle`".
```
Comment (irori) [inline subresource-loading.bs:162]: Is it okay to remove this part?

Comment (hayatoito) [reply subresource-loading.bs:162]: It seems we still need this. Undone. Thanks!

File: subresource-loading.bs [hunk 2]
```diff
@@ -171,24 +180,26 @@ changes:
        specific requirements for the error handling here, so currently an
        `error` event is fired similarly to the case of an empty `src` attribute.
 
-- Insert the following case to <a spec="html">prepare a script</a> step 27.2:
+- Insert the following case to <a spec="html">prepare the script element</a>
+  step 27.2:
```
Comment (irori) [inline subresource-loading.bs:184]: These step numbers need to be updated (again), but that could be a separate PR.

**Review by irori — APPROVED** [review]

File: subresource-loading.bs (hunk 2 above)
Comment (hayatoito) [reply subresource-loading.bs:184]: Thanks. Yeah, step numbers can be updated later. Step numbers become out-of-sync easily. :(
